### ISSUE TITLE
emergency-site.dev is down

### ISSUE THREAD

**Issue by nfriedli** [maxboeck/emergency-site#28]
The demo site (https://emergency-site.dev/) doesn’t work anymore :(

**Comment by TBG-FR** [comment]
It seems to be fixed now ;)

**Comment by nfriedli** [comment]
No. The https://emergency-site.dev/ homepage does not match the one of this repository. There is a strange advertising link in the text and a weird text.

**Comment by TBG-FR** [comment]
It contains the link to this repo many times, and the same informations as the README

https://emergency-site.dev/posts/2020-03-20-getting-started/#advanced-setup

**Comment by nfriedli** [comment]
Only @maxboeck will be able to tell us if the https://emergency-site.dev/ domain is still his and if it is official for this project.

**Comment by maxboeck** [comment]
@nfriedli @TBG-FR I do not own the domain emergency-site.dev anymore. I did initially, but it expired a year or so ago (right when I had COVID myself) and when I noticed, someone had already snatched it up. So I changed the demo URL for the project to the free Netlify domain.

Whoever owns it now apparently set up a copy of this project there, but with that advertising message. 
Apologies for the confusion!

**Comment by TBG-FR** [comment]
Thanks for the clarification @maxboeck ! 

Mea culpa @nfriedli, you were right, the fact that the new owner uses the same project confused me, that's evil smart

**Comment by nfriedli** [comment]
> So I changed the demo URL for the project to the free Netlify domain.

@maxboeck, you probably should modify the URL here: https://mxb.dev/blog/emergency-website-kit/

Thx a lot for this wonderful kit!

**Comment by maxboeck** [comment]
@nfriedli ah, missed that one. Thanks for the hint, updated!
